Mixin/Service/API/RouteAPI.swift

```diff
@@ -0,0 +1,236 @@
+import Foundation
+import Alamofire
+import CryptoKit
+import MixinServices
+
+final class RouteAPI {
+    
+    enum Error: Swift.Error {
+        case missingPublicKey
+        case missingPrivateKey
+        case encodeMessage
+        case emptyResponse
+        case calculateAgreement
+        case combineSealedBox
+    }
+    
+    static func sumsubToken(completion: @escaping (MixinAPI.Result<String>) -> Void) {
+        struct Response: Decodable {
+            let token: String
+        }
+        request(method: .get, path: "/kyc/token") { (result: MixinAPI.Result<Response>) in
+            completion(result.map(\.token))
+        }
+    }
+    
+    static func createInstrument(with token: String, completion: @escaping (MixinAPI.Result<PaymentCard>) -> Void) {
+        request(method: .post, path: "/checkout/instruments", with: ["token": token], completion: completion)
+    }
+    
+    static func instruments(completion: @escaping (MixinAPI.Result<[PaymentCard]>) -> Void) -> Request {
+        request(method: .get, path: "/checkout/instruments", completion: completion)
+    }
+    
+    static func deleteInstrument(with token: String, completion: @escaping (MixinAPI.Result<Empty>) -> Void) {
```

Comment:
Why is this method not being called anywhere ?

Reply:
Fixed in 05acdf4f0babafeeb63ec648ad4e26847f03b461